src/feature.ts

```diff
@@ -10,3 +10,9 @@ export interface FeatureInterface {
   strategies: StrategyTransportInterface[];
   variants: VariantDefinition[];
 }
+
+export interface ClientFeaturesResponse {
```

Comment:
Any help for a better name for this appreciated! It is essentially the interface for the data returned by the Unleash Client SDK API.